### PR TITLE
Update for v0.3

### PR DESCRIPTION
**Pull request by lthms**

### CODE REVIEW

File: docs/network/evm-nodes.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ You can get the kernel by importing it from a running Etherlink Smart Rollup nod
 
 ## Getting the `octez-evm-node` binary
 
-The easiest way to get the `octez-evm-node` binary is to download the binaries distributed as part of its latest release (currently [`v0.1`](https://gitlab.com/tezos/tezos/-/releases/octez-evm-node-v0.1)).
+The easiest way to get the `octez-evm-node` binary is to download the binaries distributed as part of its latest release (currently [`v0.2`](https://gitlab.com/tezos/tezos/-/releases/octez-evm-node-v0.2)).
 More precisely, we provide static binaries for Linux systems (for amd64 and arm64 architectures).
 
-As an alternative, you can use the minimal Docker image [tezos/tezos-bare:octez-evm-node-v0.1](https://hub.docker.com/layers/tezos/tezos-bare/octez-evm-node-v0.1/images/sha256-e5fdf87f2827dd971e2097faff88c6c6ce53793a2191a903a6a905a647c051f5?context=explore), which contains the correct version of the binary.
+As an alternative, you can use the minimal Docker image [tezos/tezos-bare:octez-evm-node-v0.1](https://hub.docker.com/layers/tezos/tezos-bare/octez-evm-node-v0.2/images/sha256-cf6849ee81d1ec1ffef18f78fe1439368d98b2ff21917f3503cee2d4b821413a?context=explore), which contains the correct version of the binary.
```
Comment (timothymcmackin) [inline docs/network/evm-nodes.md:21]:
```suggestion
As an alternative, you can use the minimal Docker image [tezos/tezos-bare:octez-evm-node-v0.2](https://hub.docker.com/layers/tezos/tezos-bare/octez-evm-node-v0.2/images/sha256-cf6849ee81d1ec1ffef18f78fe1439368d98b2ff21917f3503cee2d4b821413a?context=explore), which contains the correct version of the binary.
```

Comment (lthms) [reply docs/network/evm-nodes.md]: Good catch.

File: docs/network/evm-nodes.md [hunk 2]
```diff
@@ -52,6 +52,29 @@ The default is `$HOME/.octez-evm-node`.
...
 You can initialize the node from a snapshot or allow it to compute the Etherlink state from genesis, which can take a long time.
 
+### From a snapshot
+
+1. Download [an Etherlink snapshot](), and use the `octez-evm-node` to import it.
+
+   ```bash
+   wget http:// -O snapshot.gz
+   octez-evm-node snasphot import snapshot.gz --data-dir $evm_observer_dir
+
```
Comment (timothymcmackin) [inline docs/network/evm-nodes.md:62]:
```suggestion
   octez-evm-node snapshot import snapshot.gz --data-dir $evm_observer_dir
```

**Comment by timothymcmackin** [comment]
Tested on mainnet with snapshot evm-snapshot-sr1Ghq66tYK9y-2684961.gz and v0.3 and it seems to work.

**Comment by lthms** [comment]
We are planning to use a better hostname soon, but should we move forward and move as-is nonetheless?

**Comment by timothymcmackin** [comment]
> We are planning to use a better hostname soon, but should we move forward and move as-is nonetheless?

I'd say move forward and make sure to update when we get a better hostname.

**Comment by lthms** [comment]
Agreed with this.

**Review by timothymcmackin — APPROVED** [review]

**Comment by timothymcmackin** [comment]
> Agreed with this.

Ok, I've approved. Feel free to make it non-draft and go ahead and merge when you're ready.

**Pull request merged**